src/config.ts

```diff
@@ -110,6 +115,12 @@ export function parseConfig(): DeploymentsConfig | DeploymentStackConfig {
   const description = getOptionalStringInput("description");
   const tags = getOptionalStringDictionaryInput("tags");
   const maskedOutputs = getOptionalStringArrayInput("masked-outputs");
+  const cloud =
+    getOptionalEnumInput("cloud", [
```

Comment:
Do we need to update docs (readme, examples)? Or are you thinking we'd do this after it's released?

Reply:
Added #87 to track this after the release in case folks starting using this feature before it's available on `v1`.